nncf/ptq/examples/bert/bert_quantization.py

```diff
@@ -0,0 +1,179 @@
+
+    :return: A tuple (ir_model_xml, ir_model_bin) where
+        `ir_model_xml` - path to .xml file.
+        `ir_model_bin` - path to .bin file.
+    """
+    if not MODEL_DIR.exists():
+        os.makedirs(MODEL_DIR)
+
+    # Step 1: Load pre-trained model.
+    model = transformers.BertForSequenceClassification.from_pretrained(PRETRAINED_MODEL_DIR)
+    model.eval()
+
+    # Step 2: Export PyTorch model to ONNX format.
+    model_name = 'bert_base_mrpc'
+    onnx_model_path = MODEL_DIR.joinpath(f'{model_name}.onnx')
+    dummy_input = (
+        torch.ones(1, MAX_SEQ_LENGTH, dtype=torch.int64),  # input_ids
+        torch.ones(1, MAX_SEQ_LENGTH, dtype=torch.int64),  # attention_mask
+        torch.ones(1, MAX_SEQ_LENGTH, dtype=torch.int64),  # token_type_ids
+    )
+    torch.onnx.export(model,
+                      dummy_input,
+                      onnx_model_path,
+                      verbose=False,
+                      opset_version=11,
+                      input_names=['input_ids', 'attention_mask', 'token_type_ids'],
+                      output_names=['output'])
+
+    # Step 3: Run Model Optimizer to convert ONNX model to OpenVINO IR.
+    mo_command = f'mo --framework onnx -m {onnx_model_path} --output_dir {MODEL_DIR}'
+    subprocess.call(mo_command, shell=True)
+
+    # Step 4: Return path to IR model as result.
+    ir_model_xml = MODEL_DIR.joinpath(f'{model_name}.xml')
+    ir_model_bin = MODEL_DIR.joinpath(f'{model_name}.bin')
+    return ir_model_xml, ir_model_bin
+
+
+def create_val_dataset() -> datasets.Dataset:
+    """
+    Creates validation MRPC dataset.
+
+    :return: The `datasets.Dataset` object.
+    """
+    raw_dataset = datasets.load_dataset('glue', TASK_NAME, split='validation')
+    tokenizer = transformers.BertTokenizer.from_pretrained(PRETRAINED_MODEL_DIR)
+
+    def _preprocess_fn(examples):
+        texts = (examples['sentence1'], examples['sentence2'])
+        result = tokenizer(*texts, padding='max_length', max_length=MAX_SEQ_LENGTH, truncation=True)
+        result['labels'] = examples['label']
+        return result
+    processed_dataset = raw_dataset.map(_preprocess_fn, batched=True, batch_size=1)
+
+    return processed_dataset
+
+
+def validate(model: openvino.runtime.Model, data_source: datasets.Dataset) -> float:
+    """
+    Validates the model on the dataset.
+
+    :param model: A model to be validated.
+    :param data_source: Dataset for validation.
+    :return: F1 score.
+    """
+    metric = evaluate.load('glue', TASK_NAME)
+    INPUT_NAMES = [x.any_name for x in model.inputs]
+    output_layer = next(iter(model.outputs))
+    for batch in data_source:
+        inputs = [
+            np.expand_dims(np.asarray(batch[key], dtype=np.int64), 0) for key in INPUT_NAMES
+        ]
+        outputs = model(inputs)[output_layer]
+        predictions = outputs[0].argmax(axis=-1)
+        predictions, references = predictions, batch['labels']
```

Comment:
Why not just like this?
```suggestion
        references = batch['labels']
```

Reply:
Thanks! That's a good point. I fixed it.